Sources/SignHereLibrary/Commands/CreateProvisioningProfileCommand.swift

```diff
@@ -122,6 +125,7 @@ internal struct CreateProvisioningProfileCommand: ParsableCommand {
         case intermediaryAppleCertificates = "intermediaryAppleCertificates"
         case certificateSigningRequestSubject = "certificateSigningRequestSubject"
         case profileName = "profileName"
+        case autoRegenerate = "autoRegenerate"
```

Comment:
Should this just be performed in the decode using a ValidationError https://apple.github.io/swift-argument-parser/documentation/argumentparser/validation/